parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -530,7 +530,10 @@ logs_to_events() {
         --arg EventTid    "0" \
         '{Timestamp: $Timestamp, OperationId: $OperationId, Version: $Version, TaskName: $TaskName, EventLevel: $EventLevel, Message: $Message, EventPid: $EventPid, EventTid: $EventTid}'
     )
-    echo ${json_string} > ${EVENTS_LOGGING_DIR}${eventsFileName}.json
+    if [ ! -f "${EVENTS_LOGGING_DIR}${eventsFileName}.json" ]; then        
```

Comment:
I believe we'd only need to check existence of `EVENTS_LOGGING_DIR`, `>` should create the file if the parent directory exists

as a result we could simplify this all down to `mkdir -p ${EVENTS_LOGGING_DIR}` without any conditional

Reply:
updated!